### ISSUE TITLE
The SMS method should be disabled when Signal is not the SMS app

### ISSUE THREAD

**Issue by paride** [WhisperSystems/Signal-Android#7048]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description

Signal is not my SMS app, but still:
- If I long tap on the send button, I have the possibility to send messages as unencrypted SMS. I would prefer not to have the possibility to do so, even by error. I like the paradigm: if it is in Signal, it's encrypted.
- If I actually try to send an SMS from Signal, it doesn't work (the message stays in the sending status forever, with the three bouncing dots).

### Device info

**Device:** Nexus 7
**Android version:** 7.1.1 (stock)
**Signal version:** 4.11.2

### Link to debug log

I don't think it's relevant.

**Comment by FeuRenard** [comment]
See also https://github.com/WhisperSystems/Signal-Android/issues/4390#issuecomment-326755632

**Comment by p4nci** [comment]
this is intended, see https://github.com/WhisperSystems/Signal-Android/pull/5877#issuecomment-264175873

btw, this should be duplicate of #5877 :thinking:

**Comment by xyzzyxw** [comment]
I disagree, because this might be useful at times. This could be made an option though, as suggested in a comment in #4390.
